.evergreen/config/standard.yml.erb

```diff
@@ -23,6 +23,32 @@ buildvariants:
     tasks:
       - name: "test-docker"
 
+  - matrix_name: "mongo-6.0"
+    matrix_spec:
+      auth-and-ssl: noauth-and-nossl
+      ruby: ["ruby-3.1"]
+      mongodb-version: ['6.0']
+      # Only standalone until mlaunch issue is resolved with replace sets.
```

Comment:
I think it would be helpful to refer to a ticket here, if any.

Reply:
Added.